go/cmd/dolt/commands/push.go

```diff
@@ -264,18 +269,40 @@ func pushToRemoteBranch(ctx context.Context, srcRef, destRef, remoteRef ref.Dolt
 	return nil
 }
 
-func progFunc(progChan chan datas.PullProgress, stopChan chan struct{}) {
```

Comment:
these changes will no longer be applicable with the v2 push/pull.  Should revert so we don't have to deal with conflicts.